lib/spack/docs/config_yaml.rst

```diff
@@ -42,6 +42,39 @@ or with braces to distinguish the variable from surrounding characters:
 The location where Spack will install packages and their dependencies.
 Default is ``$spack/opt/spack``.
 
+---------------------------------------------------
+``install_hash_length`` and ``install_path_scheme``
+---------------------------------------------------
+
+The default Spack package installation path can be difficult to
+comprehend and if very long can also create problems for scripts with
+hardcoded shebangs. There are two parameters to help with that. The
+``install_hash_length`` parameter can set the length of the hash in
+the installation path from 0 to 32. The default uses the full 32 characters.
```

Comment:
If you agree with my comment about using a minimimum > 1 in the schema for config.yaml, could you update this doc?